src/containers/announcements/Announcements.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { Typography } from 'antd';
+import { AnnouncementsList } from '../../components/announcements-list/AnnouncementsList';
+import styled from 'styled-components';
+import { ChungusContentContainer } from '../../components';
+
+const { Title } = Typography;
+
+const Content = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const StyledTitle = styled(Title)`
+  margin: 12px;
+  margin-left: 0px;
+`;
+
+const Announcements: React.FC = () => {
+
+    React.useEffect(() => {
+        window.scrollTo(0, 0);
+    }, []);
```

Comment:
@rymaju I see that we're executing this hook in many other places too. We should consider factoring it out into a custom hook, so we can plug it in wherever we need it. Something like:

useScrollTop.tsx
```
export const useScrollTop = () => {
    React.useEffect(() => {
        window.scrollTo(0, 0);
    }, []);
} 
```
If you agree and wanna make it into a small ticket, I can pick it up.

Reply:
I agree, but I question why we need to tell the window to scroll to the top of the page every time?

Reply:
I also am pretty confused why we need to do it. I thought it would be default already.
But without this hook, the page I built is also not scrolling to the top, so I followed this strange convention that's happening in other places. We should definitely figure out what's going on so we can scrap this

Reply:
Oops, yes well by default react router (and actually the HTML browser router specs itself) push the current window location onto the stack alongside history. React router dom wont automatically scroll to top for you so you have to do it yourself :(